lib/cdm_utils.rb

```diff
@@ -3,10 +3,23 @@
 require "fileutils"
 
 module CDMUtils
+
+  def self.available_collections
+    digital_collections = DigitalCollection.all.map{ |c| c.collection_alias }
```

Comment:
seems like a great opportunity to use [pluck](http://apidock.com/rails/ActiveRecord/Calculations/pluck) here:
`DigitalCollection.pluck(:collection_alias)`